mito-ai/src/Extensions/AiChat/ChatTaskpane.tsx

```diff
@@ -141,6 +142,24 @@ const ChatTaskpane: React.FC<IChatTaskpaneProps> = ({
     const streamingContentRef = useRef<string>('');
     const streamHandlerRef = useRef<((sender: CompletionWebsocketClient, chunk: ICompletionStreamChunk) => void) | null>(null);
 
+    const updateModelOnBackend = async (model: string): Promise<void> => {
```

Comment:
Maybe in the useEffect where we call the `initializeChatHistory` function we should also call `updateModelOnBackend` to the websocket model is updated when the taskpane is opened